src/resources/dicts/fr.edn

```diff
@@ -644,9 +644,9 @@
     :flashcards/modal-toggle-preview-mode "Activer la prévisualisation"
     :flashcards/modal-toggle-random-mode "Activer le mode aléatoire"
     :flashcards/modal-welcome-desc-1 "Vous pouvez ajouter \"#card\" à un bloc pour le transformer en…"
-    :flashcards/modal-welcome-desc-2 "Vous pouvez"
+    :flashcards/modal-welcome-desc-2 "Vous pouvez "
```

Comment:
my understanding is that tongue prefers not to split things up like this, and rather to just write it in a function directly. But this seemed simplest for now :)